packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -212,6 +196,7 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
    * @param attestationsRequested The number of attestations to request
    */
   async approveAttestationFee(attestationsRequested: number) {
+    // TODO should this use the registry like we do below in the request() function?
```

Comment:
This does under the hood use the registry, but in this case we actually also need the contract instance to approve

Reply:
Will remove comment